sample-code/tasksignal-propagation.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>postTask Example: TaskSignal propagation</title>
+
+<script src='support/common.js'></script>
+
+<div id='content'></div>
+
+<script>
+  async function runExample() {
+
+    function returnCurrentTaskSignal() {
+      return scheduler.currentTaskSignal;
+    }
+
+    async function propagationEqualityAndPriorities() {
```

Comment:
Should we show off propagating through something non-postTask (like fetch), or is that difficult to do in a file://

Reply:
Sure, added